models/linkages/R/write.restart.LINKAGES.R

```diff
@@ -17,10 +17,10 @@
 ##' @return NONE
 ##' @export
 ##' 
-write.restart.LINKAGES <- function(out.dir, runid, time, settings, analysis.vec,
-                                   RENAME = TRUE, variables,
+write.restart.LINKAGES <- function(outdir, runid, time, settings, analysis.vec,
```

Comment:
it seems like arguments of `write.restart.LINKAGES` doesn't agree with how it's being called by `sda.enkf`, would that be a problem?